lib/Atat/Objects.pm

```diff
@@ -0,0 +1,96 @@
+# Domain model.
+#
+# This file describes our domain using an object model.
+#
+# The following terminology may be helpful:
+#     * 'package' defines a class.
+#     * A class 'has' an attribute, which may be another object (delegation).
+#     * Plural attributes imply a 1-many (or many-many) relationship.
+#     * A class 'is' another class (vertical composition).
+#     * A class 'does' a role (horizontal composition).
+
+# User: An ATAT user.
+package User {
+  use Mojo::Base -base;
+  has 'username';    # Username in ATAT
+  has 'piv';         # Data from PIV card.
+  has 'roles';       # Roles within ATAT.
+}
+
+# UserRole: Defines a set of permissions surrounding access to a CSP.
+package UserRole {
+  use Mojo::Base -base;
+  has 'csp_account';    # CSP account (optional)
+  has 'permissions';    # List of permissions (strings) for this role.
+}
+
+# Account: An account.
+package Account {
+  use Mojo::Base -base;
+  has 'csp_role';       # The role in the CSP.
+  has 'token';          # Auth token.
+}
+
```

Comment:
Changes to this: 

- [ ] This object should also be retitled to say "CSPAccount". 
- [ ] This object should have a Task Order number that it is referencing.  One Task Order number can be associated with multiple CSP accounts.